src/Components/3DV/SceneView.stories.tsx

```diff
@@ -16,6 +35,231 @@ export default {
     }
 };
 
+const defaultGizmoElementItems: TransformedElementItem = {
+    meshIds: ['tank6_LOD0.003_primitive0', 'tank6_LOD0.003_primitive1'],
+    parentMeshId: 'tank6_LOD0.003_primitive0'
+};
+
+export const Gizmo = () => {
+    const [state, dispatch] = useReducer(
+        ADT3DSceneBuilderReducer,
+        defaultADT3DSceneBuilderState
+    );
+    const [isGizmo, setIsGizmo] = useState<boolean>(true);
+
+    const setGizmoTransformItem = useCallback(
+        (gizmoTransformItem: TransformInfo) => {
+            dispatch({
+                type: SET_GIZMO_TRANSFORM_ITEM,
+                payload: gizmoTransformItem
+            });
+            console.log(
+                'new position: ',
+                gizmoTransformItem.position,
+                '\nnew rotation: ',
+                gizmoTransformItem.rotation
+            );
+        },
+        []
+    );
+
+    const setGizmoElementItem = useCallback(
+        (gizmoElementItem: TransformedElementItem) => {
+            dispatch({
+                type: SET_ELEMENT_TO_GIZMO,
```

Comment:
nit: just to be consistent with the action names with the one above maybe we can rename this?
```suggestion
                type: SET_GIZMO_ELEMENT_ITEM,
```

Reply:
done